config/config.go

```diff
@@ -149,7 +149,7 @@ func New(env []string) (WatchdogConfig, error) {
 		return c, fmt.Errorf("HTTP write timeout must be over 0s")
 	}
 
-	if len(c.FunctionProcess) == 0 && c.OperationalMode != ModeStatic {
```

Comment:
Why do you feel this is "wrong"?

Reply:
Both are used in the stdlib

https://stackoverflow.com/questions/18594330/what-is-the-best-way-to-test-for-an-empty-string-in-go

Reply:
I think you are right, I assumed ASM part to be different but it is the same, will rollback this gosimple rule

https://godbolt.org/z/h83vaqE1x